File: sum_diagonals.c
#include <stdio.h>
int main() {
    int n, sum = 0;
    scanf("%d", &n);
    int mat[n][n];
    for (int i = 0; i < n; i++)
        for (int j = 0; j < n; j++)
            scanf("%d", &mat[i][j]);

    for (int i = 0; i < n; i++) {
        sum += mat[i][i];
        if (i != n - 1 - i)
            sum += mat[i][n - 1 - i];
    }
    printf("Sum: %d\n", sum);
    return 0;
}

File: sum_diagonals.py
n = int(input())
mat = [list(map(int, input().split())) for _ in range(n)]
s = 0
for i in range(n):
    s += mat[i][i]
    if i != n - 1 - i:
        s += mat[i][n - 1 - i]
print("Sum:", s)
